d-rita/fastfoodfast_react
# #164353005 Set up empty react project

**What does this PR do?**
This PR creates an empty react project.

**Description of tasks completed**
- Install node and npm
- Add package.json file
- Install dependencies
- Set up webpack
- Set up Eslint

**How to manually test this branch?**
Fetch and check out this branch using `git fetch origin ch-set-up-react-project-164353005 && git checkout ch-set-up-react-project-164353005`
To install dependencies, run `yarn install`
In the terminal, use `yarn start` to run the application on localhost.

**Related Pivotal Tracker Story**
[#164353005](https://www.pivotaltracker.com/story/show/164353005)
